tools/msvs/install_tools/install_tools.bat

```diff
@@ -3,21 +3,20 @@
 setlocal
 
 cls
+
 echo ====================================================
 echo Tools for Node.js Native Modules Installation Script
 echo ====================================================
 echo.
-echo This Boxstarter script will install Python and the Visual Studio Build Tools,
-echo necessary to compile Node.js native modules. Note that Boxstarter,
-echo Chocolatey and required Windows updates will also be installed.
+echo This script will install Python and the Visual Studio Build Tools, necessary
+echo to compile Node.js native modules. Note that Chocolatey and required Windows
+echo updates will also be installed.
 echo.
 echo This will require about 3 Gb of free disk space, plus any space necessary to
-echo install Windows updates.
-echo.
-echo This will take a while to run. Your computer may reboot during the
-echo installation, and will resume automatically.
+echo install Windows updates. This will take a while to run.
 echo.
-echo Please close all open programs for the duration of the installation.
```

Comment:
Is there a reason that this part has been removed?  I think this advice is still valid.

Reply:
Do you really close any program to install a software on Windows? I for one never do, and I haven't seen anyone do that...

Reply:
While in most case there are no issues, this is always good advice to make sure there is no interference. I'll add it back.

Reply:
@aduh95 One of the issues we had raised several times on the Boxstarter repo is that users do not know that Boxstarter will be rebooting their machines and they lose their work.